docs/pages/upcoming-releases.mdx

```diff
@@ -99,9 +99,9 @@ using Teleport web UI without needing to self-host the email plugin.
 
 | Version | Date          |
 |---------|---------------|
-| 1.25.0  | December 2024 |
+| 1.26.0  | December 2024 |
 
-### 1.25.0
+### 1.26.0
```

Comment:
@r0mant should I also update expectations given the uncertainty with NetIQ?

Or should I delay it until we have a more clear picture?

Reply:
Let's update it once we have a better idea of the timeline.